vscode-extension/.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+  "root": true,
+  "ignorePatterns": "out/*",
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "extends": ["airbnb-base", "airbnb-typescript/base"],
+  "rules": {
+    "operator-linebreak": "off",
+    "@typescript-eslint/object-curly-spacing": "off",
+    "import/prefer-default-export": "off",
+    "no-await-in-loop": "off"
```

Comment:
Can wee add https://eslint.org/docs/rules/linebreak-style set to "unix"?

Maybe it's already fixed by Prettier, but when I first imported this PR I got an error from an internal linter:

<img width="542" alt="Screen Shot 2022-04-05 at 9 28 25 PM" src="https://user-images.githubusercontent.com/162735/161896066-ddcb88df-c531-4346-abf1-f81cf6d98baa.png">

Reply:
yeah I think this should be fixed with the latest & greatest